syscall_other.go

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2025 The Ebitengine Authors
+
+//go:build (android || linux) && (386 || arm)
```

Comment:
* `android` is included by `linux`, so you don't need to specify android.
* This might be problematic when compiling an app with PureGo for minor architectures like Mips. PureGo doesn't have to work for such arches, but at least this should not prevent compilation.

Thus, what about

```go
//go:build linux && !amd64 && !arm64 && !loong64
```